### ISSUE TITLE
RFC: Make workspace cloning more robust

### ISSUE THREAD

**Issue by stweil** [OCR-D/core#429]
Currently cloning of a workspace with `ocrd workspace clone --download` aborts if some files cannot be downloaded.

It would help if instead of aborting the download all other files would be finished.

Example: http://gei-digital.gei.de/viewer/metsresolver?id=PPN1024726142. Obviously the TIFF images are only available locally but not for download over the Internet.

**Comment by kba** [comment]
> It would help if instead of aborting the download all other files would be finished.

Yes and it would be in line with the recent change for mets.xml (`skip` instead of `raise`).

**Comment by kba** [comment]
> http://gei-digital.gei.de/viewer/metsresolver?id=PPN1024726142

Do you have another example? I cannot reach that one.

**Comment by stweil** [comment]
Nor can I. That looks like a temporary failure of the GEI website. So either wait, or look for other Intranda libraries - they might all be similar. I could also provide a local copy, but that will not help much as long as that website is down.

**Comment by stweil** [comment]
Here is an extract with one of the entries which cause a fatal exception:

    <mets:file ID="FILE_0028_PRESENTATION" MIMETYPE="image/tiff">
      <mets:FLocat LOCTYPE="URL" xlink:href="file:///opt/digiverso/viewer/tiff/PPN1024726142/00000028.tif"/>
    </mets:file>

**Comment by stweil** [comment]
GEI is online again. I tried several of their METS files, and they all include references to local files which of course cannot be cloned.

**Comment by kba** [comment]
OK, I can reproduce the problem, on my TODO list.
